internal/pkg/cli/env_init.go

```diff
@@ -469,7 +574,17 @@ func BuildEnvInitCmd() *cobra.Command {
   /code $ copilot env init --name test --profile default
 
   Creates a prod-iad environment using your "prod-admin" AWS profile.
-  /code $ copilot env init --name prod-iad --profile prod-admin --prod`,
+  /code $ copilot env init --name prod-iad --profile prod-admin --prod
```

Comment:
should the default examples have the `noCustomResourcesFlag` flag?

Reply:
i'll add it!